src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,62 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    static final String SUPPLY = "supply";
+    static final String BUY = "buy";
+    static final String RESULT = "result";
+    static final String SEPARATOR = ",";
+    static final String SPLITTER = " ";
```

Comment:
parameters should have access modifiers
rename some variables  
```suggestion
    static final String SUPPLY = "supply";
    static final String BUY = "buy";
    static final String RESULT = "result";
    static final String COMMA_ SEPARATOR = ",";
    static final String SPACE_SEPARATOR = " ";
```